frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
     end
 
     -- tools tab
+    if Device.isKobo() then
```

Comment:
Yes, indeed.
On May 26, 2016 8:05 AM, "Qingping Hou" <notifications@github.com> wrote:

> In frontend/apps/filemanager/filemanagermenu.lua
> <https://github.com/koreader/koreader/pull/2045#discussion_r64695368>:
>
> > @@ -112,6 +117,43 @@ function FileManagerMenu:setUpdateItemTable()
> >      end
> >
> >      -- tools tab
> > +    if Device.isKobo() then
>
> Yeah, I first started trying to move this part into device specific code.
> Then I ended up not doing it because I kind of want to avoid polluting
> device module with something we will throw away in the future soon. I still
> haven't decided on this one yet.
>
> The problem is that the menu would have to be rewritten lest menu item
> order is completely haphazard. That's already a bit of an issue.
>
> @Frenzie <https://github.com/Frenzie> , You mean it's hard to control the
> order of menu items?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/koreader/koreader/pull/2045/files/32d16ce93cdfa3fdd458b7c505dfca26e18034a1#r64695368>
>